filters-api.md

```diff
@@ -19,38 +23,80 @@ The supported filter fields as of v1.96 are:
 * `label`
 * `annotation` (same syntax as label, see examples)
 
-The supported filter ops as of v1.96 are:
-
-* `:` (equality, or "contains" if an array type)
-* `!:` (inequality, or "not contains" if an array type)
-
-Filter values are strings surrounded by `"`. Multiple values can be separated by commas `,`, representing logical OR.
-
-Each individual filter is separated by a `+`, representing logical AND.
+Added in v1.105 of Kubecost:
+* `department`
+* `environment`
+* `owner`
+* `product`
+* `team`
+
+The supported filter ops in v1.96 of Kubecost are:
+* `:` Equality
+  * For string fields (e.g. namespace): equality
+  * For slice/array fields (e.g. services): slice contains at least one value equal (equivalent to `~:`)
+  * For map fields (e.g. labels): map contains key (equivalent to `~:`)
+* `!:` Inequality, or "not contains" if an array type
+
+Added in v1.105 of Kubecost:
+* `~:` Contains
+  * For string fields: contains
+  * For slice fields: slice contains at least one value equal (equivalent to `:`)
+  * For map fields: map contains key (equivalent to `:`)
+* `!~:` NotContains, inverse of `~:`
```

Comment:
`NotContains` reads like a property or command flag. Would it make more sense to split these words?
i.e.: not contains or does not contain

Reply:
I lean towards keeping it as `NotContains` because if we ever get into a conversation about operators using anything other than the operator itself (`!~:` in this case), I'd rather see consistent (if esoteric) naming to smooth the likely-complex conversation.

I'm still open to changing it if you think it's detrimental to the effectiveness of the documentation, though! The above is an opinion I'm totally willing to relinquish.